sql/hive/pom.xml

```diff
@@ -238,36 +238,13 @@
              <configuration>
                <sources>
                  <source>v${hive.version.short}/src/main/scala</source>
-                 <source>${project.build.directory/generated-sources/antlr</source>
+                 <source>${project.build.directory}/generated-sources/antlr</source>
```

Comment:
I'll remove it, but since this patch may take some more iterations before it's ready, feel free to do it as part of another patch too.